Formula/sdl2_mixer.rb

```diff
@@ -30,14 +31,13 @@ def install
       --disable-music-flac-shared
       --disable-music-midi-fluidsynth-shared
       --disable-music-mod-mikmod-shared
-      --disable-music-mod-modplug-shared
```

Comment:
actually we want to keep this disabled so that the linkage is explicit

Reply:
I'm not sure I understand, this defines the -DMODPLUG_DYNAMIC, without this we need to compile libmodplug statically with mrboom, but we don't have a libmodplug.a to do that.

Reply:
The difference is not whether it's static or dynamic but whether the path to the dylib is explicit or not.

Reply:
With `--disable-music-mod-modplug-shared`, here's what you get:
```
iMac-TMP:~ joe$ brew linkage sdl2_mixer
System libraries:
  /System/Library/Frameworks/AudioToolbox.framework/Versions/A/AudioToolbox
  /System/Library/Frameworks/AudioUnit.framework/Versions/A/AudioUnit
  /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
  /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
  /usr/lib/libSystem.B.dylib
Homebrew libraries:
  /usr/local/opt/libmodplug/lib/libmodplug.1.dylib (libmodplug)
  /usr/local/opt/libvorbis/lib/libvorbis.0.dylib (libvorbis)
  /usr/local/opt/libvorbis/lib/libvorbisfile.3.dylib (libvorbis)
  /usr/local/opt/sdl2/lib/libSDL2-2.0.0.dylib (sdl2)
iMac-TMP:~ joe$ 
```

Reply:
But it does compile sdl2_mixer differently if you look at music_modplug.c, is this OK?

Reply:
Can you test whether `mrboom` works as expected if you compile with `--disable-music-mod-modplug-shared`?

Reply:
:+1: :-1: ?

Reply:
It works fine with SDL2_mixer --with-libmodplug

Reply:
Cool.

Reply:
So since --disable-music-mod-modplug-shared was a default it shouldn't change right ?

Reply:
correct

Reply:
And I assume you tested it without the `--with-libmodplug`?

Reply:
By which I mean, are you sure the new dependency is actually required?

Reply:
@frranck ?

Reply:
yeah sure,
https://github.com/Javanaise/mrboom-libretro/blob/master/common.cpp#L348

Reply:
ok